addon-test-support/fire-event.js

```diff
@@ -42,14 +42,12 @@ export function fireEvent(element, type, options = {}) {
   @method buildBasicEvent
   @param {String} type
   @param {Object} (optional) options
-  @param {Boolean} (optional) bubbles
-  @param {Boolean} (optional) cancelable
   @return {Event}
   @private
 */
-function buildBasicEvent(type, options = {}, bubbles = true, cancelable = true) {
+function buildBasicEvent(type, options = {}) {
   let event = document.createEvent('Events');
-  event.initEvent(type, bubbles, cancelable);
+  event.initEvent(type, true, true);
```

Comment:
If you can test it in a IE VM and it works, go for it. I'm surprised, since it's not something I can read in the spec anywhere. The only thing I see in the spec is using the `initEvent` function.